lib/screens/sign_in_page.dart

```diff
@@ -91,7 +93,9 @@ class SignInPage extends StatelessWidget {
                             fontWeight: FontWeight.w600,
                           ),
                         ),
-                        onPressed: () {},
+                        onPressed: () {
+                          context.read<AuthenticationService>().launchURLApp();
```

Comment:
This seems to lead to an infinite loading webpage on my Android phone, @Darcy382 @patel-kalp are y'all able to see if this happens on your emulators?

Reply:
@rsquared226 just tried it and it works fine for me, must just be an android thing.

https://user-images.githubusercontent.com/30302496/145095807-0c6c8cf9-1f00-4b04-a25c-c6aaf5667290.mov